packages/playground/src/components/debug-menu.ts

```diff
@@ -203,6 +217,38 @@ export class DebugMenu extends LitElement {
     this.workspace.exportYDoc();
   }
 
+  private _chooseFileToSave() {
+    if (!isFSSupported()) {
+      return window.alert('save to local is not supported!');
+    }
+    setRootDirectory().then(result => {
+      if (result?.granted) {
+        this.fsHandle = result.handle;
+        this.dialog.show();
+      }
+    });
+  }
+
+  private _saveToLocal() {
+    if (!isFSSupported()) {
+      return window.alert('save to local is not supported!');
+    }
+    const input = <SlInput>document.getElementById('input-file-name');
+    const value = input.value;
+    const filename = value.endsWith('.affine') ? value : `${value}.affine`;
+    getFileHandle(filename, this.fsHandle).then(async handle => {
+      if (!handle) {
+        handle = await createFileHandle(filename, this.fsHandle);
+      }
+      if (handle) {
+        await writeContentToFile(handle, Workspace.toLocal(this.workspace));
```

Comment:
Seems that we only serialize YDoc to disk for now?

Reply:
yep, im still working on that

Reply:
fixed